test/state/state.cpp

```diff
@@ -117,19 +117,22 @@ evmc_message build_message(const Transaction& tx, int64_t execution_gas_limit) n
 }
 }  // namespace
 
+void clear_empty_or_destructed_accounts(State& state, evmc_revision rev)
```

Comment:
![image](https://user-images.githubusercontent.com/573380/229915916-fe9a2db0-cec0-4dac-b478-b05f6e11d01a.png)

Maybe not very specific, but can we name it `finalize()`?

Reply:
Good idea :)

Reply:
This YP part is actually wrong: https://github.com/ethereum/yellowpaper/issues/885.